docs/core/deploying/index.md

```diff
@@ -17,26 +17,26 @@ When an executable is produced, you can specify the target platform with a runti
-| [framework-dependent executable](#publish-framework-dependent) for a specific platform.  |         | ✔️      | ✔️      |  ✔️     | [`dotnet publish -r <RID> --self-contained false`](../tools/dotnet-publish.md) |
-| [framework-dependent cross-platform binary](#publish-framework-dependent).               | ✔️      | ✔️      | ✔️      |  ✔️     | [`dotnet publish`](../tools/dotnet-publish.md) |
-| [self-contained executable](#publish-self-contained).                                    | ✔️      | ✔️      | ✔️      |  ✔️     | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
+| Type                                                                                     | Command                                                                        |
+|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| [framework-dependent executable](#publish-framework-dependent) for the current platform. | [`dotnet publish`](../tools/dotnet-publish.md)                                 |
+| [framework-dependent executable](#publish-framework-dependent) for a specific platform.  | [`dotnet publish -r <RID> --self-contained false`](../tools/dotnet-publish.md) |
+| [framework-dependent binary](#publish-framework-dependent).                              | [`dotnet publish`](../tools/dotnet-publish.md)                                 |
+| [self-contained executable](#publish-self-contained).                                    | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md)                        |
 
 For more information, see [.NET dotnet publish command](../tools/dotnet-publish.md).
 
 ## Produce an executable
 
-Executables aren't cross-platform. They're specific to an operating system and CPU architecture. When publishing your app and creating an executable, you can publish the app as [self-contained](#publish-self-contained) or [framework-dependent](#publish-framework-dependent). Publishing an app as self-contained includes the .NET runtime with the app, and users of the app don't have to worry about installing .NET before running the app. Apps published as framework-dependent don't include the .NET runtime and libraries; only the app and 3rd-party dependencies are included.
+Executables aren't cross-platform, they're specific to an operating system and CPU architecture. When publishing your app and creating an executable, you can publish the app as [self-contained](#publish-self-contained) or [framework-dependent](#publish-framework-dependent). Publishing an app as self-contained includes the .NET runtime with the app, and users of the app don't have to worry about installing .NET before running the app. Publishing an app as framework-dependent doesn't include the .NET runtime; only the app and 3rd-party dependencies are included.
```

Comment:
Acrolinx wouldn't like "publishing an app" but if you prefer it it's fine with me.